ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/OperationPool.java

```diff
@@ -72,4 +79,8 @@ public void addAll(SSZList<T> items) {
   public void removeAll(SSZList<T> items) {
     operations.removeAll(items.asList());
   }
+
+  public Set<T> getAll() {
+    return operations;
```

Comment:
I'd suggest 
```suggestion
    return Collections.unmodifiableSet(operations);
```

to be paranoid.

Reply:
Done.